src/github-issues-prs.ts

```diff
@@ -394,7 +398,9 @@ export class GitHubIssuesPrsProvider implements TreeDataProvider<TreeItem> {
 			try {
 				const { stdout } = await exec('git remote -v', { cwd: folder.uri.fsPath });
 				for (const url of new Set(allMatches(/^[^\s]+\s+([^\s]+)/gm, stdout, 1))) {
-					const m = /[^\s]*github\.com[/:]([^/]+)\/([^ ]+)[^\s]*/.exec(url);
+					// const m = /[^\s]*github\.com[/:]([^/]+)\/([^ ]+)[^\s]*/.exec(url);
+					const m = new RegExp(`[^\s]*${this.getURL()}[/:]([^/]+)\/([^ ]+)[^\s]*`).exec(url);
```

Comment:
Backslashes need escaping in the string. Slashes don't. The URL might need some escaping too. Maybe (please check):
```ts
const m = new RegExp(`[^\\s]*${this.getURL().replace(/\./g, '\\.')}[/:]([^/]+)/([^ ]+)[^\\s]*`).exec(url);
```